src/app/ruler/ruler.js

```diff
@@ -146,5 +153,18 @@ document.querySelector('.ruler__inner__theme').addEventListener('click', () => {
 window.addEventListener('resize', updateMesures);
 
 ipc.on('settings-changed', loadSettings);
+ipc.on('toggle-center-guides', toggleCenterGuides);
+
+function toggleCenterGuides() {
+	showCenterGuides = !showCenterGuides;
+
+	if (showCenterGuides === true) {
```

Comment:
Do we have to check for `true`, or would `truthy` be enough here?
e.g. `if (showCenterGuides)`